GitUI/CommandsDialogs/RefLogDialog/BranchNameForm.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Windows.Forms;
+
+namespace GitUI.CommandsDialogs.RefLogDialog
+{
+    public partial class BranchNameForm : Form
```

Comment:
>Branch name should be normalized according to the settings.

Done.

>Is it possible to use the existsing Create branch dialog here?

I'm not really convinced by that idea. This dialog is too complex for just creating a branch in this specific case (I myself not very confident on the result if you begin to change some options in the form before creating the branche, in the context of using the reflog).

I have made the development in another branch ( https://github.com/pmiossec/gitextensions/tree/reflog_reuse_create_branch_form ) if you want to test. Let me know...